dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -252,7 +252,7 @@
         }
         if (CollectionUtils.isNotEmpty(this.registries)) {
             this.registries.forEach(registryConfig -> {
-                if (registryConfig.getScopeModel() != applicationModel) {
+                if (registryConfig != null && registryConfig.getScopeModel() != applicationModel) {
```

Comment:
**Suggestion**: Add null check for registryConfig to avoid potential NullPointerException. <br> **Code Suggestion**: 
 ```
 +                if (registryConfig != null && registryConfig.getScopeModel() != applicationModel) {
 ```